modules/virt-creating-offline-vm-snapshot-cli.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-offline-vm-snapshots.adoc
+
+[id="virt-creating-offline-vm-snapshot-cli_{context}"]
+= Creating an offline virtual machine snapshot in the CLI
+
+You can create a virtual machine (VM) snapshot for an offline VM by creating a `VirtualMachineSnapshot` object.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Power down the VM for which you want to create a snapshot.
+
+.Procedure
+
+. Create a YAML file to define a `VirtualMachineSnapshot` object that specifies the name of the new `VirtualMachineSnapshot` and the name of the source VM.
++
+For example:
++
+[source,yaml]
+
+----
+apiVersion: snapshot.kubevirt.io/v1alpha1
+kind: VirtualMachineSnapshot
+metadata:
+  name: my-vmsnapshot <1>
+spec:
+  source:
+    apiGroup: kubevirt.io
+    kind: VirtualMachine
+    name: my-vm <2>
+----
+<1> The name of the new `VirtualMachineSnapshot` object.
+<2> The name of the source VM.
+
+. Create the `VirtualMachineSnapshot` resource. The snapshot controller creates a `VirtualMachineSnapshotContent` object, binds it to the `VirtualMachineSnapshot` and updates the `status` and `readyToUse` fields
+of the `VirtualMachineSnapshot` object.
++
+[source,terminal]
+----
+$ oc create -f <my-vmsnapshot>.yaml
+----
+
+.Verification steps
+
+* Verify that the `VirtualMachineSnapshot` object is created and bound with `VirtualMachineSnapshotContent`. The `readyToUse` flag must be set to `true`.
++
```

Comment:
We should add a step to verify that the expected disks are included in the snapshot.

Reply:
@aglitke Can you please elaborate? Is there a specific field in the `VirtualMachineSnapshot` object configuration that the user should look at to verify that the expected disks are included in the snapshot?

Reply:
According to Michael you can check the volumeBackups property in the VMSnapshotContent resource associated with the VMSnapshot.